app/controllers/admins/panel_controller.rb

```diff
@@ -10,6 +10,22 @@ def index
     @drivers = Driver.order('active DESC').all
   end
 
+  def edit_driver_photo
+    @driver = Driver.find_for_authentication(id: params[:id])
+    if request.post?
```

Comment:
Do you ever [REST](https://en.wikipedia.org/wiki/Representational_state_transfer)? :face_with_head_bandage:
Please use the different actions or, at least, write some specs for this controller.
Also, you have RuboCop offense here: [Use a guard clause instead of wrapping the code inside a conditional expression](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals).